README.md

```diff
@@ -60,23 +44,54 @@ gem 'faraday', '~> 0.8'
 
 For the Faraday middleware to have the correct trace ID, the rack middleware should be used in your application as explained above.
 
-Then include ZipkinTracer::FaradayHandler as a Faraday middleware:
+Then include `ZipkinTracer::FaradayHandler` as a Faraday middleware:
+
+```ruby
+require 'faraday'
+require 'zipkin-tracer'
+
+conn = Faraday.new(:url => 'http://localhost:9292/') do |faraday|
+  faraday.use ZipkinTracer::FaradayHandler, 'service_name' # 'service_name' is optional (but recommended)
+  # default Faraday stack
+  faraday.request :url_encoded
+  faraday.adapter Faraday.default_adapter
+end
+```
+
+Note that supplying the service name for the destination service is optional;
+the tracing will default to a service name derived from the first section of the destination URL (e.g. 'service.example.com' => 'service').
+
+## Tracers
```

Comment:
Maybe add something at the top explaining that you need to choose between one of the three following